src/context/AppProvider/hooks/useAccountsFetchQuery.tsx

```diff
@@ -43,7 +30,7 @@ export const useAccountsFetchQuery = () => {
           chainId,
           wallet,
           isSnapInstalled,
-          checkConnectedAccountIds: false, // don't check connected account ids, we're detecting runtime support for chains
```

Comment:
revert comment?

Reply:
fml, reverted in [0917dd0](https://github.com/shapeshift/web/pull/9727/commits/0917dd05ff9b6c3553e43ff082b3915fe9081dee)